ArcGISRuntimeSDKQt_CppSamples/Scenes/Add3DTilesLayer/README.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "category": "Scenes",
+    "description": "Add a layer to visualize 3D tiles data that conforms to the OGC 3D Tiles specification.",
```

Comment:
Yeah, I was not sure when writing the sample. I took the `dataItems` field out because we didn't know what we were going to do with this large IM dataset.  Now I added it back and left `path` and `itemId` blank.